src/Backend/Compile.hs

```diff
@@ -40,6 +42,22 @@ supportedLlvmVersions = makeVersion . (: [minorVersion]) <$> supportedMajorVersi
   where minorVersion = 0
         supportedMajorVersions = [maxLlvmVersion, maxLlvmVersion - 1 .. minLlvmVersion]
 
+-- | llvm-config is not available in current LLVM distribution for windows, so we
+-- need use @clang -print-prog-name=clang@ to get the full path of @clang@.
+--
+-- We simply assume that @clang.exe@ already exists in @%PATH%@.
+--
+clangBinPath :: String -> IO FilePath
+clangBinPath command = trim <$> checkClangExists trySuffixes
+  where trySuffixes = "" : fmap (('-' :) . showVersion) supportedLlvmVersions
+        checkClangExists (suffix : xs) =
+          handle (\(_ :: IOException) -> checkClangExists xs)
+          $ readProcess (command ++ suffix) ["-print-prog-name=" ++ command ++ suffix] ""
```

Comment:
Do Clang installations for Windows add a version suffix like the Ubuntu packages? (The binary is called e.g. `clang-5.0`.)

Otherwise this might be redundant work.

Reply:
Fixed.